src/jsMain/kotlin/baaahs/jsx/MosiacUI.kt

```diff
@@ -0,0 +1,14 @@
+@file:JsModule("js/MosiacUI.jsx")
+@file:JsNonModule
+
+package baaahs.jsx
+
+import baaahs.SheepSimulator
+import react.RProps
+
+@JsName("default")
+external class MosiacUI(props: Props) {
```

Comment:
I think it's spelled 'mosaic' but I'm not particularly particular. ;-)

Reply:
Haha, I spelled it wrong so many times and thought I caught all of them. I'll fix this!